frontend/src/components/Humanboard.vue

```diff
@@ -165,7 +169,7 @@ export default {
         this.pieceColor = this.gameInfo.players.player2Color;
       }
       this.opponent = this.opponentName();
-      this.time = this.gameInfo.timeToGo * 60000;
+      this.time = this.gameInfo.timeToGo * 600;
```

Comment:
600? I suppose it should be 60 000

Reply:
Sorry. I was experimenting with the game end by timeout and reduced this value. It must be sure 60000.